Lecture2/src/task4/util/MarathonHelper.java

```diff
@@ -0,0 +1,42 @@
+package task4.util;
+
+import task4.model.MarathonParticipant;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class MarathonHelper {
+
+    private MarathonHelper() {
+    }
+
+    public static List<MarathonParticipant> createParticipants(String[] names, int[] time) {
+        int length = 0;
+        List<MarathonParticipant> participants = new ArrayList<>();
+        if (names.length != time.length) {
+            System.out.println("The name array doesn't equals time array!!\n" +
+                    "The length will be equals lover value!");
+            if (names.length > time.length) {
+                length = time.length;
+            }
+        } else {
+            length = names.length;
+        }
+        for (int i = 0; i < length; i++) {
+            participants.add(new MarathonParticipant(names[i], time[i]));
+        }
+        return participants;
+    }
+
+    public static void printParticipants(List<MarathonParticipant> participants) {
+        int placeCounter = 1;
+        for (MarathonParticipant participant : participants) {
+            System.out.printf("Place [%-2d] - %s\n", placeCounter, participant);
+            placeCounter++;
+        }
+    }
+
+    public static void printParticipantByPlace(List<MarathonParticipant> participants, int index) {
```

Comment:
method printParticipantByPlace - expect place as param  - not index

Reply:
more over you should get first Participant (get(0)) when  place is 1

Reply:
fix, rewrite this methods